content/2018-12-18_dvanacty-sraz-matice.md

```diff
@@ -0,0 +1,85 @@
+Title: Dvanáctý sraz - matice
+Date: 2018-12-18 18:00:00
+Modified: 2018-12-18 18:00:00
+Author: Iveta Česalová
+
+<div class="text-justify">Po počátečním zmatku, kdy si Petr zapomněl svůj počítač, jsme poslední sraz roku zahájily opět rekapitulací, co kdo dělal. Doposud byla hra rozdělena na čtyři moduly – backend, frontend, game a testy. Nyní je v backendu pouze třída Robot, State a funkce pro načtení souřadnic, cesty k robotům, počtu políček a počáteční stav hry.</div>
```

Comment:
Co je to `<div class="text-justify">?

Reply:
zarovnání textu do bloku, ne vlevo

Reply:
Aha! Já jsem si říkala, proč to někde vidím blbě, ale říkala jsem si, že to bude nějaký problém u mě, když je to všude jinde ok a je to všude označené stejně. Zobrazoval se mi špatně odstavec "Nekombinovat více knihoven". Je možné, že se to vypně jen někde?

Reply:
Markdown není dělaný na prezentaci (co je kurzívou, co je jakým fontem, co je jak zarovnané, ...), ale na obsah (co je zvýrazněné, co je odstavec, co je ukázka kódu, ...). Když je potřeba něco, co Markdown neumí, dá se použít HTML, ale mělo by se do HTML přepnout úplně.

> Je možné, že se to vypně jen někde?

Jo. Předpokládám že to má něco do činění s koncovým `</div>`, ale detaily neznám.

Reply:
právěže to nejde na celém blogu protože to vypíná ty věci v markdownu. ano už vidím odstavec Nekombinovat více knihoven opraveno.
Jestli to mám všude smazat tak řeknite, jen jsem chtěla aby to trochu vypadalo.
Opraveno snad všechno.

Reply:
Mrknu na to odpoledne, jak se dostanu k soukromému počítači

Reply:
Nastavit to pro celý blog jde, ale zjistit který soubor upravit není vůbec přímočaré (což je moje chyba – mohlo to být líp popsáno v dokumentaci nástroje, který jsem vám zvolil).
Dělá se to takhle: https://github.com/PyLadiesCZ/roboprojekt-blog/pull/29

Reply:
tak nevím jestli to funguje to #29 Musí se s tím něco ještě dělat?

Reply:
Do bloku se to podle mě nezarovnává, ale markdown teď už funguje všude hezky,

Reply:
Jsem skoro pro to teď vyhodit ven, ať je to tam před další lekcí, a tohle upravit dodatečně. @ivet1987 pokud jsi pro, dej vědět a já to schválím.

Reply:
@anezkamll prosím o schválení :)

Reply:
> tak nevím jestli to funguje to #29 Musí se s tím něco ještě dělat?

Nefunguje :(
V #30 by to mělo být opravené.

Reply:
Díky moc @encukou, tohle už mi fungovalo hezky, mergla jsem